added quotes to hyphenated words

ajchu28 opened the pull request:
Here is the map and popup code on a feature branch with the JS separated and the html/css put into the correct files.

ajchu28 committed:
added quotes to hyphenated words

nbriz commented:
@ajchu28 i just merged @yruann 's PR, i think now would be a good time to pull her changes into ur feature branch
```
git pull upstream main
```
&& then integrate the code u've written in ur `frontend/mapScript.js` into her `frontend/js/map.js` script (&& then removing ur `mapScript.js`

ajchu28 committed:
Merge branch 'main' of https://github.com/net-art-uchicago/cta-file-sharing into mapview

ajchu28 committed:
merged files with Ying

ajchu28 commented:
Changes made in merge: 

- Deleted my mapView.html and mapScript.js files
- Reconfigured map.js so popups are created from a JSON object
- Added popups as a source, then loaded icons in a layer. Popups display when the icon is clicked on. Reasoning is on Mapbox GL JS page [Add Markers](https://docs.mapbox.com/help/getting-started/add-markers/) in the "Add Data" section, which says that this method is better for performance, especially for maps which will display >20 markers.
